rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -29,6 +29,100 @@
 import ctypes
 
 
+class ROCALNumpyIterator(object):
+    def __init__(self, pipeline, device="cpu", device_id=0, return_max_roi=False):
+        self.loader = pipeline
+        self.device = device
+        self.device_id = device_id
+        self.output_memory_type = self.loader._output_memory_type
+        self.output_list = None
+        self.batch_size = self.loader._batch_size
+        self.return_max_roi = return_max_roi
```

Comment:
what is the use of this variable?

Reply:
This variable is set if we want to return the buffer with max ROI dims instead of max_shape dims